jinahub/indexers/query/vector/FaissIndexer/README.md

```diff
@@ -2,64 +2,3 @@
 
```

Comment:
I would expect a more extensive `README.md`

Reply:
We haven't decided on a README structure yet, have we? This should be done when we decide